src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -506,7 +506,7 @@ def update_progress(cur_loss, iteration):
         'training_epochs': training_iterations * batch_size // num_images,
         'training_iterations': training_iterations,
         'max_iterations': max_iterations,
-        'training_loss': progress['smoothed_loss'],
+        'training_loss': progress['smoothed_loss']
```

Comment:
Nit: let's avoid mutating this line in this PR unnecessarily. Before, there was a trailing comma here.

(This way, we keep the `git blame` history for this line simpler.)